src/cmake/openvino.cmake

```diff
@@ -48,6 +48,7 @@ target_link_libraries(${TARGET_NAME} PRIVATE openvino::reference
                                              openvino::pugixml
                                              ${CMAKE_DL_LIBS}
                                              Threads::Threads
+                                             ${GRAPH_COMPILER_LIBS}
```

Comment:
Does graph compiler require explicit LLVM dependencies below, or GC already carries all the stuff inside?

Reply:
It doesn't, but for switching the modes in runtime, MLIR_ALL_LIBS could still be required here?

Reply:
Right, for the dynamic switch, it is still required. To measure binary size more accurately, is it easy to implement GC-only mode?

Reply:
I mean, for example, if we are not passing MLIR_DIR to cmake, then GC-only mode is activated. Not sure how GC is related to MLIR libs/headers.

Reply:
MLIR_DIR is also required for GC.